packages/terra-status/tests/wdio/status-spec.js

```diff
@@ -0,0 +1,39 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('medium');
+
+describe('Status', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/raw/tests/status/status-default'));
```

Comment:
nit: each of these beforeEach can be changed to before since we do not need to refresh the page between test steps